spec/helpers/view_helper_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+feature "View helpers", type: :helper do
+  describe "#govuk_link_to" do
```

Comment:
These are in govuk_components so we don't need to test.

Reply:
Removed.